app/views/comments/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>New Comment</h2>
+
+<%= form_with model: [@post, Comment.new], url: user_post_comments_path(@post.user, @post), local: true do |form| %>
+  <%= form.label :text, "Add a Comment:" %>
+  <%= form.text_area :text %>
+  <%= form.submit "Submit Comment" %>
+<% end %>
```

Comment:
- [ ] Good Job for Implemented  form in the` views/comments/new.html.erb` file which accepts input for creating comments. It's working as expecting as show with the below screenshot.
![image](https://github.com/EvansSnave/Blog_app_rails/assets/93598189/b866c656-2735-498f-8408-68ff304a3b23)
![image](https://github.com/EvansSnave/Blog_app_rails/assets/93598189/e2a9f0ca-07da-4573-87e1-75828a960cf0)